.circleci/config.yml

```diff
@@ -81,6 +81,30 @@ jobs:
       - run:
           name: Publish package
           command: "./node_modules/.bin/lerna publish --message 'chore: release new versions' --yes"
+  generate-repo:
+    <<: *defaults
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/
+      - add_ssh_keys:
+          fingerprints:
+            - '95:74:63:db:82:da:b3:e6:57:0c:3a:bd:3f:ff:ce:c6'
+      - run:
+          name: Authenticate with registry
+          command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+      - run: git config --global user.email circleci@circleci
+      - run: git config --global user.name CircleCI
+      - run: 
+          name: Publish init output to GitHub
+          command: |
+            cd packages/create
+            git clone git@github.com:open-wc/init-output-test.git
+            yarn start --type scaffold --scaffoldType wc --features testing demoing building demoing --tagName init-output-test --scaffoldFilesFor testing demoing building --installDependencies false --writeToDisk true --typescript no --overwriteFile always
+            cd init-output-test
+            git add .
+            git commit -am 'updated npm init @open-wc output'
+            git push -f
```

Comment:
This might "fail" in some cases when there are no changes... do we want to let it fail or do some more in-depth management here?

Reply:
Does it mark the CI as failed? That would be a shame.

Reply:
We could make it an "optional" step in GitHub so that it didn't fail the build...but it does go red in Circle... let me see what a `|| exit 0` does!

Reply:
Oh, that did it!